### DISCUSSION TITLE
Containers under gluetun can't access other machines on the LAN

### DISCUSSION THREAD

**Discussion by ceramicv** [qdm12/gluetun#2228] [Q&A]
I have a bunch of containers running with gluetun in a docker-compose file. I want those apps to be able to send alerts/notifications to another machine in my network that runs Gotify. 

However after setting the notifications up they alweays failed and when I opened the containers terminals and tries pinging the host name and/or IP address of my gotify machines there is never any response. 

I'm guessing the gluetun network is not allowing this and I tried using network_mode: "service:gluetun" as well as "container:gluetun" but neither made a difference. I also tried adding the machine to an extra_hosts section under the gluetun service in docker-compose.yml but no joy there either.

Anyone know a way round this?

**Comment by kilbouri** [comment]
I'm certainly no expert but I don't think what you seek is feasible. Gluetun's express purpose is to run all traffic through a VPN tunnel, including DNS. This means that any traffic that happens on Gluetun's network is, effectively, happening on some remote server. That remote server cannot talk to devices on your local network.

About all you could reasonably do without changing how your stack works is making the services that need to receive notifications publicly accessible. This is, obviously, dangerous and for that reason I don't think what you seek is feasible.

**Comment by PrinceFrosty** [comment]
`FIREWALL_OUTBOUND_SUBNETS:`?

**Reply by jdwake** [reply]
This is the way. Thank you!

https://github.com/qdm12/gluetun-wiki/blob/main/setup/options/firewall.md
